app/api/police-area.js

```diff
@@ -0,0 +1,77 @@
+const config = require('../config')
+const wreck = require('@hapi/wreck')
+const { getPoliceForces } = require('../api/ddi-index-api/police-forces')
+
+const osBaseUrl = config.osPlacesApi.baseUrl
+const policeBaseUrl = config.policeApi.baseUrl
+const postcodeEndpoint = 'postcode'
+const policeLocationEndpoint = 'locate-neighbourhood'
+
+const osPlacesOptions = {
+  headers: {
+    key: config.osPlacesApi.token
+  },
+  json: true
+}
+
+const policeApiOptions = {
+  json: true
+}
+
+const getPostcodeLongLat = async (postcode) => {
```

Comment:
Could you move this into the os-places module and import it? Just in case we need to reuse it anywhere and will make our lives easier if we need to mock.